src/client/src/orangehrmRecruitmentPlugin/pages/VacancyList.vue

```diff
@@ -0,0 +1,174 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div class="orangehrm-background-container">
+    <oxd-grid ref="scrollerRef" class="orangehrm-container">
+      <oxd-grid-item v-for="vacancy in vacancies" :key="vacancy">
+        <vacancy-card
+          :vacancy-description="vacancy.vacancyDescription"
+          :vacancy-id="vacancy.vacancyId"
+          :vacancy-title="vacancy.vacancyTitle"
+        ></vacancy-card>
+      </oxd-grid-item>
+      <oxd-loading-spinner
+        v-if="isLoading"
+        class="orangehrm-container-loader"
+      />
+    </oxd-grid>
+  </div>
+  <div>
+    <div class="orangehrm-container-img-div">
```

Comment:
```suggestion
    <div class="orangehrm-vacancy-list-footer">
```

Reply:
if I change it, the infinite scroll is not working properly.

Reply:
you need to debug why

Reply:
done